packages/nextjs/app/builders/0x9dF97939e2aA22954CFddcD6a57008DA5d5F38Be/page.tsx

```diff
@@ -0,0 +1,52 @@
+import Image from "next/image";
+import Link from "next/link";
+import { faGithub } from "@fortawesome/free-brands-svg-icons";
+import { faTwitter } from "@fortawesome/free-brands-svg-icons";
+import { faTelegram } from "@fortawesome/free-brands-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import type { NextPage } from "next";
+
+const Profile: NextPage = () => {
```

Comment:
Rename your component to something more specific to you, since there will be multiple `Profile` components in the project.